Library/Homebrew/cli/named_args.rb

```diff
@@ -88,6 +97,12 @@ def load_formula_or_cask(name, only: nil, method: nil)
           begin
             return Cask::CaskLoader.load(name, config: Cask::Config.from_args(@parent))
           rescue Cask::CaskUnavailableError => e
+            default_cask_tap = Tap.default_cask_tap
```

Comment:
Does the `e` provide `e.cask.tap` or something that can be used to see if it's the default tap?

Reply:
No.

Reply:
Cool, thanks 👍🏻